frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -107,6 +112,11 @@ const UpdatesAvailableAlert = ({cv}) => <div className="alert alert-info">
   </Button>
 </div>;
 
+const FeatureUnavailableAlert = () => <div className="alert alert-info">
+  <i className="pficon pficon-info"></i>
+  Cluster update feature is unavailable for the current version.
```

Comment:
I'd suggest

```suggestion
  Cluster updates are not supported in beta 1.
```

just to make it really clear. The current message sounds almost like they are just no updates available.